package.json

```diff
@@ -65,7 +65,7 @@
     "nodemon": "^1.12.1",
     "postcss-loader": "0.13.0",
     "prettier": "^1.5.3",
-    "puppeteer": "^1.2.0",
+    "puppeteer": "^1.5.0",
```

Comment:
is this still needed in main package file?

Reply:
yep, for the web scraping

Reply:
i mean - does it have to be in root package.json

Reply:
removed from root package.json